packages/tracing/src/span.ts

```diff
@@ -307,9 +307,9 @@ export class Span implements SpanInterface, SpanContext {
     description?: string;
     op?: string;
     parent_span_id?: string;
-    sampled?: boolean;
     span_id: string;
     start_timestamp: number;
+    status?: string;
```

Comment:
We can use typeof `SpanStatus` here and for `apm`, unfortunately it's not defined for utils though.

Reply:
Probably yes, will update.

Reply:
Hmm we have status typed as string in a bunch of places, including `SpanStatus | string`. I'll leave fixing that to another PR and focus on the bigger picture here.